openmetadata-ui/src/main/resources/ui/src/locale/languages/fr-fr.json

```diff
@@ -473,24 +506,31 @@
     "service-email-required": "Un Email pour le Service account est requis",
     "special-character-not-allowed": "Les charactères spéciaux ne sont pas authorisés",
     "sql-query-tooltip": "Requête avec 1 ligne ou plus entrainera l'échec du test.",
+    "stage-file-location-message": "Temporary file name to store the query logs before processing. Absolute file path required.",
     "successfully-completed-the-tour": "Vous avez fini la visite avec succès.",
     "sure-to-remove": "Etes-vous sûr de vouloir supprimer le",
     "team-no-asset": "Votre equipe n'a pas de resources de données",
+    "thread-count-message": "Set the number of threads to use when computing the metrics. If left blank, it will default to 5.",
```

Comment:
Did we plan to add keys with en lang by default?

Reply:
I have added key, so we don't missout in the file